repo/packages/A/alluxio-enterprise/3/resource.json

```diff
@@ -0,0 +1,59 @@
+{
+  "assets": {
+    "uris": {
+      "alluxio-agent": "https://alluxio-dcos-release.s3.amazonaws.com/agent",
+      "bootstrap-zip": "https://alluxio-dcos-release.s3.amazonaws.com/artifacts/2.1.0-1.7.0/bootstrap.zip",
+      "custom-download-url": "",
```

Comment:
@ryadav88 This is how it is supposed to be?

@jsancio This failed in the package.dcos generation CI.

Reply:
Also the url at `"alluxio-agent": "https://alluxio-dcos-release.s3.amazonaws.com/agent"` is not public (and I believe it is not an asset). Cc @jsancio

Reply:
@madanadit  ^

Reply:
yes, this is the way its supposed to be. in an offline env, `custom-download-url` will be used to download the alluxio enterprise tarball which is not hosted publicly. when online, `alluxio-agent` is the prefix for a platform specific go binary which can authenticate using a license and download the alluxio enterprise binary. the path `https://alluxio-dcos-release.s3.amazonaws.com/agent` is a prefix to assets but not an asset itself (as pointed out).

Reply:
Okay since this is already in universe, I wanted to let you know that this may not be desirable in a future revision of this package. You should be able to use mustache temptation and config.json to get the required business logic. We added a new CI that would bundle the packages in to a single file (as .dcos) which would help us run the packages in airgapped clusters. The style of URLs above broke that CI.